Casks/m/morisawa-desktop-manager.rb

```diff
@@ -0,0 +1,39 @@
+cask "morisawa-desktop-manager" do
+  version "2.1.2"
+  sha256 "6cbfb743ff4f66b571eb41561c050812ba0314bbe1c3ea544138d68052023484"
+
+  url "https://morisawafonts.com/resources/dm/#{version}/mac/site/MorisawaDesktopManager_#{version}.dmg"
+  name "Morisawa Desktop Manager"
+  desc "Manager for Morisawa Fonts"
+  homepage "https://morisawafonts.com/mypage/dm/"
```

Comment:
The current homepage is redirecting to a login page.

```suggestion
  homepage "https://en.morisawafonts.com/"